doc/supported-platforms.md

```diff
@@ -2,11 +2,11 @@
 
 Ignition is currently only supported for the following platforms:
 
-* [Bare Metal] - Use the `coreos.config.url` kernel parameter to provide a URL to the configuration. The URL can use the `http://`, `https://`, `tftp://`, or `s3://` schemes to specify a remote config or the `oem://` scheme to specify a local config, rooted in `/usr/share/oem`.
-* [PXE] - Use the `coreos.config.url` and `coreos.first_boot=1` (**in case of the very first PXE boot only**) kernel parameters to provide a URL to the configuration. The URL can use the `http://`, `https://`, `tftp://`, or `s3://` schemes to specify a remote config or the `oem://` scheme to specify a local config, rooted in `/usr/share/oem`.
+* [Bare Metal] - Use the `ignition.config.url` kernel parameter to provide a URL to the configuration. The URL can use the `http://`, `https://`, `tftp://`, or `s3://` schemes to specify a remote config or the `oem://` scheme to specify a local config, rooted in `/usr/share/oem`.
+* [PXE] - Use the `ignition.config.url` and `ignition.firstboot=detected` (**in case of the very first PXE boot only**) kernel parameters to provide a URL to the configuration. The URL can use the `http://`, `https://`, `tftp://`, or `s3://` schemes to specify a remote config or the `oem://` scheme to specify a local config, rooted in `/usr/share/oem`.
```

Comment:
Should be `ignition.firstboot=1` for PXE.

Reply:
As `coreos.first_boot=1` is consumed by bootengine, I cut the details on this one. One can click the link to see the Cl-specific PXE options. We should probably create FCOS specific docs soon.

Reply:
I'm not seeing a link there?  I agree that including the parameter name in the docs is a layering violation, but it's better to be helpful.

Reply:
the markdown is processed in some way, the `[PXE]` link is defined further down in the file

Reply:
@bgilbert: see https://coreos.com/ignition/docs/latest/supported-platforms.html

Reply:
Whoops, sorry, you're right.  I'd still rather see the docs specify exactly what to do, since that part is confusing enough as is.

Reply:
@bgilbert I originally suggested him to drop that part, as that parameter has now become distro-dependent (`coreos.` on CL and `ignition.` in other places). I believe we'd better leave that out of this list and add a dedicated "first-boot mechanism" doc/section instead. That goes in the direction of decoupling CL (OS) and Ignition docs for the longer term.

Reply:
`ignition.firstboot` is technically distro-dependent, but I think it'd be too confusing for different distros to choose different parameter names.  So maybe the right thing is to leave `coreos.first_boot` in the 2.x docs and change it to `ignition.firstboot` after branching for 3.x.